src/mpags-cipher.cpp

```diff
@@ -70,6 +38,8 @@ int main(int argc, char* argv[])
             << "                   Stdin will be used if not supplied\n\n"
             << "  -o FILE          Write processed text to FILE\n"
             << "                   Stdout will be used if not supplied\n\n"
+            << "  -k KEY [0, 25]   Key for the Caesar Cipher (default: 0)\n\n"
+            << "  -d               Decrypt flag"
```

Comment:
Very good that you're updating the help message as you go.

Another source of documentation, the README file in the src directory probably should be updated at this point to describe the new build procedure and new functionality.